modules/claims_api/db/migrate/20200617105905_add_signature_errors_default_to_claims_api_power_of_attorney.rb

```diff
@@ -0,0 +1,5 @@
+class AddSignatureErrorsDefaultToClaimsApiPowerOfAttorney < ActiveRecord::Migration[6.0]
```

Comment:
If this is a migration fix for something that needed to be fixed in the last migration then why isn't it under the claims_api module namespace?

Reply:
Its not really a fix, one of the linter/cop/danger modules doesnt allow a default to be set on a new column. so this is the follow up migration to do that. Now sure I follow what you mean about it not being in the claims_api namespace though

Reply:
Disregard, I thought this had been stored in the db/migrate instead of the modules/claims_api/db/migrate folder.